STYLE_GUIDE.md

```diff
@@ -78,15 +78,14 @@ generic:
 
 -   Standard library imports (most generic)
 -   Third-party package dependencies
--   Voxel51-authored non-FiftyOne package dependencies
 -   FiftyOne modules (least generic)
 
 For core FiftyOne imports, we import modules as `fox`, where `x` is the first
 letter of the module imported. If necessary, we use `foxy` to disambiguate
 between two modules that start with the same letter.
 
 Within each import group, imports should be sorted alphabetically by full
-package path, ignoring `from` and `import`.
+package path, with direct `import`s first and `from` imports after.
```

Comment:
because this is how all the automated tools handle it, and we weren't really doing a good job of following this guidance anyway

Reply:
[configurable](https://pycqa.github.io/isort/docs/configuration/options.html#force-sort-within-sections)